src/mmw/sass/_bootstrap.scss

```diff
@@ -5,12 +5,12 @@
 // Reset and dependencies
 @import "bootstrap/normalize";
 @import "bootstrap/print";
-@import "bootstrap/glyphicons";
+// @import "bootstrap/glyphicons";
```

Comment:
No big deal. I just make a habit out of removing code rather than commenting it out. If it makes your life easier in the long run, keep it.